src/Microsoft.DotNet.Build.Tasks.Feed/project.json

```diff
@@ -5,9 +5,10 @@
     "System.Security.Cryptography.Algorithms": "4.2.0",
     "System.Reflection.Metadata": "1.3.0",
     "NETStandard.Library": "1.6.0",
-    "Newtonsoft.Json": "9.0.1",
-    "NuGet.Versioning": "4.3.0-preview2-4095",
-    "NuGet.Packaging": "4.3.0-preview2-4095"
+    "Newtonsoft.Json": "10.0.3",
```

Comment:
Should we upgrade everything to 10.0.3 rather than just the Feed project? I'm not sure if there are any concrete problems, but it seems odd to depend on two versions.

Reply:
Yeah I did think about doing that but did not know if there were hard dependencies on a particular version. In my case, Sleet depends on 10.0.3. I'll upgrade all.

Reply:
This is going to cause issues because it is higher then the version that lives in the CLI we are using. Can we avoid bumping the version?

Reply:
@weshaggard  Sleet has a hard dependency on 10.0.3. I did try updating in the code base but it fails to build. How hard would it be to update CLI's?

Reply:
That will be a large undertaking. Plus we want to stick with the 2.0.0 RTM CLI for most if not all of this release.

Reply:
Starting on Sleet 2.0.0 where a lot of perf improvements were done a dependency on 10.x was introduced to use Json.NET async pattern. Moving back to 9.0.1 will require several code changes and most likely some perf trade offs. @weshaggard what do you suggest would be a good middle ground?